### PR TITLE
[CAMEL-14400] Add support for Weka Data Mining

### PR DESCRIPTION
**Pull request by tdiesler**

### CODE REVIEW

File: components/camel-weka/src/main/java/org/apache/camel/component/weka/WekaComponent.java [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+@Component("weka")
+public class WekaComponent extends DefaultComponent {
+
+    @Override
+    protected Endpoint createEndpoint(String urispec, String remaining, Map<String, Object> params) throws Exception {
+
+        // Init the configuration
+        WekaConfiguration config = new WekaConfiguration(this);
+        setProperties(config, params);
```
Comment (omarsmak) [inline components/camel-weka/src/main/java/org/apache/camel/component/weka/WekaComponent.java]: I recall, you should do `setProperties` on config from the endpoint. Perhaps either @oscerd or @davsclaus can confirm this.

Comment (oscerd) [reply components/camel-weka/src/main/java/org/apache/camel/component/weka/WekaComponent.java]: Yes, you should this on the endpoint, so we could avoid reflections.

Comment (davsclaus) [reply components/camel-weka/src/main/java/org/apache/camel/component/weka/WekaComponent.java]: Yes correct

**Comment by oscerd** [comment]
I'll merge this tomorrow

**Comment by oscerd** [comment]
Merged on master. Thanks

**Pull request closed without merging**